root/layout/components/sidebar/SidebarDataQuality.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {QUALITY_NAMES} from '../../../static/scripts/common/constants';
+import {addColon, l} from '../../../static/scripts/common/i18n';
+
+import {SidebarProperty} from './SidebarProperties';
+
+type Props = {|
+  +quality: QualityT,
+|};
+
+const SidebarDataQuality = ({quality}: Props) => {
+  const name = QUALITY_NAMES.get(quality);
+  return name ? (
+    <SidebarProperty className="data-quality" label={addColon(l('Data Quality'))}>
```

Comment:
That changed message, for the best.

Reply:
Yes, I saw we already had a `Data Quality` string, though I now see it's from the statistics domain. (Not sure if Transifex suggest translations from different domains.)